content/docs/introduction/faq.md

```diff
@@ -25,7 +25,10 @@ The main Prometheus server runs standalone and has no external dependencies.
 Yes, run identical Prometheus servers on two or more separate machines.
 Identical alerts will be deduplicated by the [Alertmanager](https://github.com/prometheus/alertmanager).
 
-The Alertmanager cannot currently be made highly available, but this is a goal.
+For high availability of the Alertmanager, you can run multiple instances in a
+[Mesh cluster](https://github.com/weaveworks/mesh) and configure the Prometheus
```

Comment:
I'd suggest linking to the relevant bit of the AM readme rather than to mesh itself